### ISSUE TITLE
How does Facebook know which site I'm browsing?

### ISSUE THREAD

**Issue by UMLAUTaxl** [velazcod/Tinfoil-Facebook#115]
I browsed getgrover.com through IceCatMobile on my smartphone and now I get advertisements for that when I open Tinfoil for Facebook. Is that my mistake? I thought TFB would protect me from that.

**Comment by jfmcbrayer** [comment]
Depends...have you ever logged in to Facebook with that IceCatMobile? If so, then their tracking cookies across the web will follow you whatever you subsequently use to log in to Facebook. 

Alternatively, they may be using another means of fingerprinting you...maybe just your home WiFi IP address.

Or it could just be a coincidence. My most seen ad on Facebook is Amazon premium beauty products, and I have absolutely no idea why 🤣

**Comment by UMLAUTaxl** [comment]
No, I never logged into FB with my browser. It can't be a coincidence since the day after I browsed that site FB showed its ad. Although before that ads never really came to my attention.
Strangely enough tracking shouldn't be possibe via IceCatMobile since an add-on named Spyblock should be included. But I can't find it in the add-on list. In Settings is an option for DNT and "Tracking protection" which are enabled since I use this app.
I activated privacy.trackingprotection.enabled in about:config. It seems to help after browsing that site and opening TFB again. I just hope that some cookies are still stored in my browser.
(If browsers are the problem what does TFB really protect us from? Is TFT somehow better in that aspect? I guess not ... how?)
